app/models/user.rb

```diff
@@ -269,20 +269,6 @@ def mpi_add_person_proxy
     mpi.add_person_proxy
   end
 
-  def mpi_add_person_implicit_search
```

Comment:
Oh yeah, good find. I think I started calling MPI directly cause I didn't want it to cache things, didn't realize we still had the functions in the User model